tests/check_check_master.c

```diff
@@ -943,11 +943,13 @@ void record_failure_line_num(int linenum)
    */
    linenum += 1;
 
-  to_write = snprintf(string, sizeof(string), "%d\n", linenum);
-  if(to_write == 0)
+  chars_printed = snprintf(string, sizeof(string), "%d\n", linenum);
+  if(chars_printed <= 0 || (size_t) chars_printed >= sizeof(string))
   {
     fprintf(stderr, "%s:%d: Error in call to snprintf:", __FILE__, __LINE__);
     exit(1);
+  } else {
```

Comment:
small point, the {} in the file follow this convention:

if() 
{
// stuff
}

Probably this else is not needed, though, as the if() bails out. Probably the to_write assignment can be outside of the else block.

Reply:
I moved it outside the else block. It's clearer that way. Thank you.